internal/ingress/controller/store/store.go

```diff
@@ -597,9 +594,26 @@ func (s *k8sStore) extractAnnotations(ing *extensions.Ingress) {
 	key := k8s.MetaNamespaceKey(ing)
```

Comment:
The comment seems to be outdated - it does not describe what the function does.

Also should we call this function `syncIngress` now?

Reply:
done